src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    operate(callback, value) {
+      callback(value);
```

Comment:
Callback function in the operate method does not have any effect on the result. You should assign the return value of the callback to the result

Reply:
If you fix it as you say, then the tests do not pass, or I misunderstood you.
![image](https://github.com/mate-academy/js_advanced_calculator/assets/124064854/10018526-15b5-48c6-96d6-b1e71c57a9ca)